roles/nginxplus/files/conf/http/researchdata_staging.conf

```
@@ -4,6 +4,7 @@ proxy_cache_path /data/nginx/researchdata-staging/NGINX_cache/ keys_zone=researc
 upstream researchdata-staging {
     zone researchdata-staging 64k;
     least_conn;
+    # server prds-staging1.princeton.edu resolve;
     server prds-staging1.princeton.edu resolve;
```

We want the load balancer to talk to staging2, right?

```suggestion
    server prds-staging2.princeton.edu resolve;
```